ovh/resource_vps_gen.go

```diff
@@ -370,6 +390,22 @@ func (v *VpsModel) MergeWith(other *VpsModel) {
 	}
 }
 
+func (v *VpsModel) ToOrder() *OrderModel {
+	return &OrderModel{
+		Order:         v.Order,
+		OvhSubsidiary: v.OvhSubsidiary,
+		Plan:          v.Plan,
+		PlanOption:    v.PlanOption,
+	}
+}
+
+func (v *VpsModel) ToInstallOptions() *InstallOptionsModel {
+	return &InstallOptionsModel{
+		ImageId:      v.ImageId,
+		PublicSSHKey: v.PublicSSHKey,
```

Comment:
not sure if this works when `v.PublicSSHKey` is nil since it will be marshalled as `null` and I don't know if this is accepted buy the API (see https://github.com/ovh/terraform-provider-ovh/blob/master/ovh/types/string.go#L40)

Maybe add a test to check :)